src/pages/Home/Formatters/forecastMarginsDataFormatter.ts

```diff
@@ -27,8 +27,15 @@ const getDemandForecastData = (demandData: string[]): DataTableDataType => {
 };
 
 export const getFormattedForecastMarginsData = (
-  rawData: ForecastMarginsNoticeAndDemandResponseData
+  rawData: ForecastMarginsNoticeAndDemandResponseData | undefined
```

Comment:
Should use rawData? here rather than | undefined

Reply:
ForecastMarginsNoticeAndDemandResponseData is an interface, so not sure if we can do this? Or I don't understand what you mean...

Reply:
Jump on a call after this so we can discuss?